syntax = "proto3";

message InventoryItem {
  int32 id = 1;
  int32 product_id = 2;
  int32 quantity = 3;
  string status = 4;
}
